languages/julia/exercises/concept/matrices-introduction/.docs/instructions.md

```diff
@@ -0,0 +1,95 @@
+Something something dot matrix display.
```

Comment:
nit: what is a dot matrix display?

Reply:
Low resolution monochrome displays, typically just a grid of green LEDs. Often seen in public transport settings or hobbyist electronics.

Reply:
Here's an example of a basic one: https://youtu.be/yBIXeHHAhrI?t=375

Reply:
Dot matrix Display:  https://www.mouser.com/ProductDetail/Adafruit/3211?qs=byeeYqUIh0MkEV1wTTyO2w%3D%3D&gclid=EAIaIQobChMImdOP5_bF6gIV0RZ9Ch2JDA-OEAQYBSABEgLDlfD_BwE

Reply:
Sweet. Thanks for the explanation and links y'all!